inc/inventorynetworkequipmentlib.class.php

```diff
@@ -117,11 +118,12 @@ function updateNetworkEquipment($a_inventory, $items_id) {
 
       // * NetworkEquipment fusion (ext)
       $db_networkequipment = array();
-      $query = "SELECT *
-         FROM `".  getTableForItemType("PluginFusioninventoryNetworkEquipment")."`
-         WHERE `networkequipments_id` = '$items_id'";
-      $result = $DB->query($query);
-      while ($data = $DB->fetch_assoc($result)) {
+
+      $params = [
+         'FROM'  => getTableForItemType("PluginFusioninventoryNetworkEquipment"),
+         'WHERE' => ['networkequipments_id' => $items_id]
+      ];
+      foreach ($DB->request($params) as $data) {
```

Comment:
Maybe avoid calling a method in the loop. I'm not sure a `foreach` is OK here; you should have used somtehing like `while ($data = $iterator->next())`